### PR TITLE
feature(content types): Add support to edit content types if cuustom …

### PR DESCRIPTION
**Pull request by ioanpocol**
…vocabularies are defined

### CODE REVIEW

**Comment by petrjasek** [comment]
is there any related ui PR ?

**Comment by ioanpocol** [comment]
Not yet

File: apps/content_types/content_types.py [hunk 1]
```diff
@@ -105,13 +129,67 @@ def _set_updated_by(self, doc):
...
+                    schema[field] = deepcopy(DEFAULT_SCHEMA[field])
+        else:
+            editor = doc['editor'] = deepcopy(DEFAULT_EDITOR)
+            schema = doc['schema'] = deepcopy(DEFAULT_SCHEMA)
+            default = True
+
+        # process custom fields defined on vocabularies
+        vocabularies = get_resource_service('vocabularies').find({})
+        for vocabulary in vocabularies:
+            if 'service' not in vocabulary:
```
Comment (petrjasek) [inline apps/content_types/content_types.py]: this filtering could be done via `find`

File: apps/content_types/content_types.py [hunk 2]
```diff
@@ -105,13 +129,67 @@ def _set_updated_by(self, doc):
     def _set_created_by(self, doc):
         doc['created_by'] = get_user_id()
 
+    def get_allowed_list(self, schema):
```
Comment (petrjasek) [inline apps/content_types/content_types.py]: does it always return `['subject']` or something else too?

Comment (ioanpocol) [inline apps/content_types/content_types.py]: This is to access some parts of the schema definition for subject and the 'subject' name is not changed (if changed also the mongodb field will change).

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**